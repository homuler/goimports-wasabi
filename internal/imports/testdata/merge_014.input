// merge empty import declarations
package main

// doc comment
import ()

import (/* postlparen comment */)