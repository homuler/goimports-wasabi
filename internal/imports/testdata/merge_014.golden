// merge empty import declarations
package main

// doc comment
import ( /* postlparen comment */ )
